llvm/lib/CodeGen/MachineLICM.cpp

```diff
@@ -597,14 +599,6 @@ void MachineLICMBase::HoistRegionPostRA(MachineLoop *CurLoop,
     const MachineLoop *ML = MLI->getLoopFor(BB);
     if (ML && ML->getHeader()->isEHPad()) continue;
 
-    // Conservatively treat live-in's as an external def.
-    // FIXME: That means a reload that're reused in successor block(s) will not
```

Comment:
As we removed this fixme (and extended the implementation accordingly), I think this can be well received by the community.